imagedephi/base_rules.yaml

```diff
@@ -3,6 +3,9 @@ description: These are the default rules for each potential piece of redactable
   (metadata, image) to be used as a fallback if there are no user defined rules for
   a piece of redactable material.
 tiff:
+  associated_images:
+    default:
+      action: delete
```

Comment:
@manthey @brianhelba I imagine it will be a bit trickier to come up with rules for associated images, as different formats will each implement this in a different way. Does it make sense to have a default action to take on associated images? 

For example, in aperio files, the image description for an IFD can be used to determine if the image is a macro or label image (but not thumbnail). If we're processing aperio files that, for some reason, have associated images that we can't consistently identify, how should those be handled? Currently I'm taking a naive approach with the "default" rule, but I'm open to other ideas about this.

Reply:
large_image has a bunch of heuristics to determine the appropriate label of an associated image.  Some readers (openslide) report this (mostly); some, like tiff, require that we scrape the information from the various ways different writers do things; others, like bioformats, may not expose enough information to actually determine the appropriate label.

In general, we will probably have some heuristics that will work some of the time.  When we don't know, the best we can probably do is have "image-1", "image-2", etc. without knowing what they are.  So, yes, generally, this means we will want a rule for otherwise unspecified associated images that would be delete or keep.